db/seeds.rb

```diff
@@ -19,9 +19,47 @@
 Packaging.create(name: 'Paper bag')
 Packaging.create(name: 'Recyclable')
 
-Certification.create(name: 'Halal')
+@halal = Certification.create(name: 'Halal')
 Certification.create(name: 'Kosher')
 Certification.create(name: 'Non-GMO')
 
 Nutrition.create(name: 'Superfood')
 Nutrition.create(name: 'High in protein')
+
+
+@lorem = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+"
+
+@vendor_picture = "http://image.shutterstock.com/image-vector/vector-illustration-supermarket-grocery-store-260nw-518613133.jpg"
+
+@first_vendor = Vendor.create(name: 'First Vendor', picture: @vendor_picture, address: "", facebook: "https://www.facebook.com/DangFoods/?", twitter: "", instagram: "", story: @lorem, mission: @lorem)
```

Comment:
I would add a Twitter or instagram link to at least one or two vendors, just to showcase the functionality.